ajax/libs/cachep2p/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "cachep2p",
+  "filename": "cachep2p.min.js",
+  "description": "The Peer to Peer Cache for the Masses",
+  "version": "0.1.5",
+  "author": {
+    "name": "Carlos Guerrero",
+    "email": "carlos@carlosguerrero.com",
+    "url": "https://cachep2p.com"
+  },
+  "homepage": "https://cachep2p.com",
+  "keywords": [
+    "cache",
+    "p2p",
+    "web2web",
+    "cachep2p",
+    "p2pcache",
+    "web2webcache",
+    "cacheweb2web",
+    "bittorrent",
+    "bittorrent client",
+    "download",
+    "mad science",
+    "peer-to-peer",
+    "peers",
+    "streaming",
+    "swarm",
+    "torrent",
+    "web torrent",
+    "webrtc",
+    "webrtc data",
+    "webtorrent"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/guerrerocarlos/cachep2p.git"
+  },
+  "npmName": "cachep2p",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "cachep2p*.+(js|css|map)"
```

Comment:
By the way, you can also just use `cachep2p*` as there is no files with the prefix that we don't want to have.